### DISCUSSION TITLE
Global Campus Offers suddenly expired, unable to redeem benefits as usual

### DISCUSSION THREAD

**Discussion by rust-kotlin** [community/community#72816] [GitHub Education]
### Hi everyone,

I applied for a student permit in March. I received the benefits on the same afternoon. However, on August 30th of this year (2023), the benefits suddenly became invalid. I am unable to access any student benefits and cannot submit any new applications.
![tmp4065](https://github.com/community/community/assets/105690584/d7b402c5-bbcb-420b-8e7b-ab6d8b29b1a6)
I attempted to resubmit my information and reapply for the Global Gamepus privileges, but the system does not allow me to do so because I already have student eligibility.
![tmp59CC](https://github.com/community/community/assets/105690584/c424ad12-819a-47d6-92ed-178c599b7726)
It shows that "Your Student Developer Pack will be ready soon!" for almost 2 months. So I am confused about what I should do to regain my Student Developer Pack, since I can't reverify my student identity. What should I do to restore my Global Gamepus privileges?
![tmpBBCE](https://github.com/community/community/assets/105690584/b946ae10-da0a-43da-93de-3016563222f3)

**Comment by rust-kotlin** [comment]
Can anyone answer my question?

**Reply by morganersery1** [reply]
Our records indicate that your coupon has been applied to your account. Can you confirm if you are now able to redeem your benefits?
